service/src/main/scala/org/apache/celeborn/server/common/http/HttpRequestHandler.scala

```diff
@@ -66,6 +66,8 @@ class HttpRequestHandler(
         service.getWorkerInfo
       case "/lostWorkers" if service.serviceName == Service.MASTER =>
         service.getLostWorkers
+      case "/blacklistedWorkers" if service.serviceName == Service.MASTER =>
```

Comment:
What do you think if we put lostWorkers and blacklistedWorkers in "/workerInfo" http request?

Reply:
> What do you think if we put lostWorkers and blacklistedWorkers in "/workerInfo" http request?

Current worker info too much information. Sometimes we only want `lostWorkers` or `blacklistedWorkers`.

But I think it's ok to add both this information to `/workerInfo` and only keep these two url